compile_clang.sh

```diff
@@ -2,8 +2,9 @@
 
 _cross_compile=$(realpath "$ANDROID_ROOT/prebuilts/gcc/linux-x86/aarch64/aarch64-linux-android-4.9/bin/aarch64-linux-android-")
 _cross_compile_32=$(realpath "$ANDROID_ROOT/prebuilts/gcc/linux-x86/arm/arm-linux-androideabi-4.9/bin/arm-linux-androideabi-")
-_clang_path=$(realpath "$ANDROID_ROOT/prebuilts/clang/host/linux-x86/")
-_clang_path=$(ls -1d "$_clang_path/clang-r"*/bin | tail -1)
+# Get the clang version from current AOSP documentation
+_clang_version=$(grep -A1 'grep -A1 '^\* \[\*\*Android Linux Kernel' $ANDROID_ROOT/prebuilts/clang/host/linux-x86/README.md | awk '/^  \* Currently clang-/ {print $NF}')
```

Comment:
I think you intended this to be:

```suggestion
_clang_version=$(grep -A1 '^\* \[\*\*Android Linux Kernel' $ANDROID_ROOT/prebuilts/clang/host/linux-x86/README.md | awk '/^  \* Currently clang-/ {print $NF}')
```
?

Otherwise:
```
/data/aosp-r/oot/compile_clang.sh: line 6: unexpected EOF while looking for matching `''
```
And obviously grepping a stringified comment doesn't make a whole lot of sense in this context :)